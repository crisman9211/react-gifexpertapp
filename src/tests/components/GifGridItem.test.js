import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { GifGridItem } from '../../components/GifGridItem';

describe(('Test GifGridItem'),()=>{
    it(('should render correctly'),()=>{
        const title = 'prub Title'
        const url = 'https://media4.giphy.com/media/W79wfYWCTWidO/200w.webp?cid=ecf05e47mp09ja27m0vm7376roucymx59cst0ahcwpjsapbl&rid=200w.webp&ct=g'
        const wrapper = shallow(
        <GifGridItem 
            title={title}  
            url={url}
        />)
        expect(wrapper).toMatchSnapshot()
    })
})