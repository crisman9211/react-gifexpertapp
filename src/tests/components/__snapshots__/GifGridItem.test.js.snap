// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test GifGridItem should render correctly 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="prub Title"
    src="https://media4.giphy.com/media/W79wfYWCTWidO/200w.webp?cid=ecf05e47mp09ja27m0vm7376roucymx59cst0ahcwpjsapbl&rid=200w.webp&ct=g"
  />
  <p>
     
    prub Title
     
  </p>
</div>
`;
